app/scripts/lib/transaction/util.ts

```diff
@@ -66,8 +84,45 @@ export async function addDappTransaction(
 export async function addTransaction(
   request: AddTransactionRequest,
 ): Promise<TransactionMeta> {
-  const { waitForSubmit } = request;
+  ///: BEGIN:ONLY_INCLUDE_IF(blockaid)
+  const {
+    transactionParams,
+    transactionOptions,
+    ppomController,
+    securityAlertsEnabled,
+    chainId,
+  } = request;
+
+  if (securityAlertsEnabled && SUPPORTED_CHAIN_IDS.includes(chainId)) {
+    try {
```

Comment:
```suggestion
  if (securityAlertsEnabled && SUPPORTED_CHAIN_IDS.includes(chainId)) {
    const {
      transactionParams,
      transactionOptions,
      ppomController,
      securityAlertsEnabled,
      chainId,
    } = request;

    try {
```

Reply:
my b. We spoke about this and 2 props are needed for the if statement. please ignore this comment